index.bs

```diff
@@ -1121,6 +1143,9 @@ When this operation is invoked, the authenticator must perform the following pro
 - Prompt the user for consent to create a new credential. The prompt for obtaining this consent is shown by the authenticator
     if it has its own output capability, or by the user agent otherwise. If the user denies consent, return an error code
     equivalent to NotAllowedError and terminate the operation.
+- If the {{ScopedCredentialOptions/verificationGesture}} parameter is set to true, prompt the user for performing an 
```

Comment:
The caveat that pressing a button doesn't count belongs here too, since this is the point implementers will actually be looking at when they decide whether they should allow pressing a button to count.